Makefile

```diff
@@ -69,18 +78,38 @@ fix: ## Fix linting issues in the code
 	${DOCKER_COMPOSE} run --rm app npm run lint:fix
 
 test: export TARGET = dev
-test: ## Run the tests
+test: ## Run all the tests
+	$(MAKE) db
+	$(MAKE) wait-healthy
+	$(MAKE) initdb
 	${DOCKER_COMPOSE} run --rm app npm run test
 
+unit-test: export TARGET = dev
+unit-test: ## Run the unit tests
+	${DOCKER_COMPOSE} run --rm app npm run test:unit
+
+integration-test: export TARGET = dev
+integration-test: ## Run the integration tests
+	$(MAKE) db
+	$(MAKE) wait-healthy
+	$(MAKE) initdb
+	${DOCKER_COMPOSE} run --rm app npm run test:integration
+
 run:
 	${DOCKER_COMPOSE} up --abort-on-container-exit --exit-code-from app; ${DOCKER_COMPOSE} down
 
 dev: export TARGET = dev
 dev: ## Build and runs the container for development
 	$(MAKE) --jobs=4 install build stop
+	$(MAKE) db
+	$(MAKE) wait-healthy
+	$(MAKE) initdb
 	$(MAKE) run
```

Comment:
Related to https://github.com/libero/article-store/pull/116/files#r371827242, I guess the `docker-compose down` inside `run` does stop the `db` container?